frontend/lib/util/format.test.ts

```diff
@@ -16,4 +18,18 @@ describe("Format util", () => {
     expect(formatNumber(input)).equals(expected);
     expect(deformatNumber(expected)).equals(parseInt(input, 10));
   });
+
+  const today = new Date();
+  today.setHours(10, 20);
+  const yesterday = new Date(today);
+  yesterday.setDate(today.getDate() - 1);
+  const one_week_ago = new Date(today);
+  one_week_ago.setDate(today.getDate() - 7);
+  test.each([
+    [today, `${t("today")} 10:20`],
+    [yesterday, `${yesterday.toLocaleString(t("date_locale"), { weekday: "long" })} 10:20`],
+    [one_week_ago, `${one_week_ago.toLocaleString(t("date_locale"), { day: "numeric", month: "short" })} 10:20`],
```

Comment:
I'd prefer hard-coding the expected values here, so there's no need to interpret the code that's now there.

Reply:
Yes but it is based on today, so the weekday and date I cannot hardcode :cry: